resources/stage_2_de-bloat/programs_to_target_by_name.txt

```diff
@@ -1,222 +1,222 @@
-%%180Search%%
-%%555%%
-%%AdBlocknWatch%%
-%%AppToU%%
-%%BlocckkTheAds%%
-%%ClipGenie%%
-%%CoolWebSearch%%
-%%CoolWWW%%
-%%Cydoor%%
-%%Coupon%%
-%%Esuack%%
-%%FashionLife%%
-%%Freenpro25%%
-%%Gamevance%%
-%%iBryte%%
-%%iStart123%%
-%%MapsGalaxy%%
-%%SaferSurf%%
-%%Savings%%
-%%SaveForYou%
-%%Search%%
-%%Shop_and_Up%%
-%%Shopper%%
-%%SpeedUpMyPC%%
-%%Start Search%%
-%%TidyNetwork%%
-%%Toolbar%%
-%%Trial%%
-%%Virtumundo%%
-%%VirusProtectPro%%
-%%WeatherBug%%
-%%WhenUsave%%
-%%Zango%%
-180 Solution%%
+*180Search*
```

Comment:
Can you update this file to revert to the old-style wildcards? (unless your program accepts both, then we can go with astericks since it's more intuitive)